pkg/ast/spl/spl.peg

```diff
@@ -951,6 +956,301 @@ TcOptionCMD <- option:("usenull" / "useother" / "nullstr" / "otherstr") {
     return optionStr, nil
 }
 
+SpanLenOption <- number:(FloatAsString / IntegerAsString) timeScale:(TimeScale)? {
+    if timeScale != nil {
+        num, err := strconv.ParseInt(number.(string), 10, 64)
+        if err != nil {
+            return nil, fmt.Errorf("SpanLength: Invalid num (%v): %v", number.(string), err)
```

Comment:
nit: in the log message change `num` to `number` so it's slightly more clear

Reply:
addressed in the latest commit.